e2e/tools/validator/src/validator/stresser/__init__.py

```diff
@@ -75,3 +75,14 @@ def run_script(self, script_path):
             start_time=start_time,
             end_time=end_time,
         )
+    
+    def run_command(self, command):
+        self.connect()
+        _, stdout, stderr = self.ssh_client.exec_command(command)
+        exit_status = stdout.channel.recv_exit_status()
+        result = stdout.read().decode('ascii').strip("\n")
+        
+        if exit_status != 0:
+            print(f"Something went wrong in running command: ", command)
```

Comment:
It is better to return an Object that holds exit_code, stdout and stderr and let the caller decide what to do than ignore this.